src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -568,5 +580,17 @@ private void CheckSize(int countToAdd)
                 }
             }
         }
+
+        private struct EncodingPreamblePair
+        {
+            public readonly Encoding Encoding;
+            public readonly byte[] Preamble;
+
+            public EncodingPreamblePair(Encoding encoding, byte[] preamble)
```

Comment:
Ah, yes, thanks! (I didn't notice the possible simplification because I was originally using `KeyValuePair<K,V>`). Fixed.